addons/website_forum/static/src/js/tours/website_forum.js

```diff
@@ -60,7 +60,7 @@ odoo.define("website_forum.tour_forum", function (require) {
         trigger: ".modal-header button.close",
         auto: true,
     }, {
-        trigger: ".o_wforum_validate_toggler[data-karma=\"20\"]:first",
+        trigger: ".o_wforum_validate_toggler[data-karma]:first",
```

Comment:
I don't see in your changes why we need to change this. I probably miss something. 😬

Reply:
Probably because, as [pointed out by jke](https://github.com/odoo/odoo/pull/74353#issuecomment-893341723) this PR was removing the karma restriction and everything was available with 0 karma with the PR

Reply:
Initially I changed it in the data by mistake, but now I've moved to demo. But the change here is still required for the test to pass